web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Upgrading and Updating",
+		href: "/Upgrading_and_Updating/index-foreman-deb.html"
+	},{
+		title: "Deploying Foreman on AWS",
+		href: "/Deploying_on_AWS/index-foreman-deb.html"
+	},{
+		title: "Configuring Smart Proxies with a Load Balancer",
+		href: "/Configuring_Load_Balancer/index-foreman-deb.html"
+	},{
+		title: "Provisioning Guide",
+		href: "/Provisioning_Guide/index-foreman-deb.html"
+	},{
+		title: "Configuring Foreman to use Ansible",
+		href: "/Configuring_Ansible/index-foreman-deb.html"
+	},{
+		title: "Managing Hosts Guide",
+		href: "/Managing_Hosts/index-foreman-deb.html"
+	},{
+		title: "Administering Foreman",
+		href: "/Administering_Red_Hat_Satellite/index-foreman-deb.html"
+	},{
+		title: "Application Centric Deployment",
+		href: "/Application_Centric_Deployment/index-foreman-deb.html"
```

Comment:
This renders as empty to me.

Reply:
@maximiliankolb I would expect this to be a fully working combination? Do you want to fix this? I might have accidentally hidden this, sorry in that case just remove the if statement from the master.adoc:

https://docs.theforeman.org/nightly/Application_Centric_Deployment/index-foreman-deb.html

Reply:
There are currently no packages to install ACD plugin on Debian/Ubuntu. This may change in the future, but IMHO we can safely not link it for now.